### PR TITLE
Updating team page on the website 

### PR DESCRIPTION
**Pull request by bahar-ap**
## Description

tidying up the team page with new pictures and up to dated details about each member

## Screenshots

https://user-images.githubusercontent.com/81912681/156330913-46e891e5-a835-4a3d-bff2-01ace77418be.mov

## Steps to Test

checkout to my branch and run yarn develop

### CODE REVIEW

**Review by kdav108 — APPROVED** [review]
LGTM! Great work!

**Review by Blake-Haydon — APPROVED** [review]
Looks great. The video was awesome in the PR!

**Review by hallgchris — APPROVED** [review]
Good stuff!

File: src/markdown/team.md [hunk 1]
```diff
@@ -121,11 +121,11 @@ subteam:
         degree: "Arts and Commerce"
         position: Team Member
         photo: ../images/Geoffrey_Meng.jpg
-      - linkedIn: https://www.linkedin.com/in/harry-blackburn-2427651b6/
```
Comment (kdav108) [inline src/markdown/team.md]: Have we missed Harry?

Comment (bahar-ap) [reply src/markdown/team.md]: Ah whoops I did, I'll fix that nonw

**Review by kdav108 — APPROVED** [review]

**Pull request merged**